Makefile

```diff
@@ -137,16 +137,25 @@ libsodium:
 	$(MAKE) clean; \
 	$(MAKE)
 
-shadowsocks: libsodium openssl
+shadowsocks_linux: libsodium openssl
 	@echo "Building shadowsocks"
 	cd shadowsocks-rust; \
 	unset CARGO_TARGET_DIR; \
-	SODIUM_STATIC=1 \
```

Comment:
`SODIUM_STATIC` is deprecated. Now they just check the existence of `SODIUM_SHARED` instead: https://github.com/sodiumoxide/sodiumoxide/blob/master/libsodium-sys/build.rs#L32